public-api/src/main/java/org/pmiops/workbench/publicapi/DataBrowserController.java

```diff
@@ -307,6 +335,7 @@ public ConceptAnalysis apply(ConceptAnalysis ca) {
 
     @Override
     public ResponseEntity<DbDomainListResponse> getSurveyList() {
+        CdrVersionContext.setCdrVersionNoCheckAuthDomain(defaultCdrVersionProvider.get());
```

Comment:
So the idea is with every api call you just call this first?   If this api call gets moved to common with this same code just work?

Reply:
I don't think this controller should moved to common; controllers in workbench generally require that the user is signed in, and use the user's credentials to determine what data they have access to. That said, yes, if you moved this to common it should still work I believe.

All of these calls should be replaced by a call that will accept a CDR version ID parameter, look up that CDR version, and call CdrVersionContext.setCdrVersionNoCheckAuthDomain with it. That will allow for CDR version switching. But I didn't want to do that in this PR.

Reply:
Cool. Yeah, sorry I just meant like on an individual endpoint basis ... like some endpoints may be exactly the same in workbench or public. So the common controller would be good that could be merged with the individual controllers. Not really sure how that would be implemented.